src/main/java/rx/Completable.java

```diff
@@ -1992,7 +2014,7 @@ public final void subscribe(CompletableSubscriber s) {
                 throw new NullPointerException("The RxJavaPlugins.onSubscribe returned a null Subscriber");
```

Comment:
I think this can only happen if `s` is `null`, which is checked in `requiredNonNull(s)`. Same in `subscribe(Subscriber)`